packages/zmarkdown/README.md

```diff
@@ -1,161 +1,243 @@
+```
+npm install zmarkdown
+```
 
-All endpoints return `[contents, metadata, messages]` as JSON.
+## Getting start
 
-* `contents` - string, the rendered HTML or LaTeX
-* `metadata` - object, depends on request options
-* `messages` - string[], info/debug/errors from parsers, plugins, compilers etc
+ 1. Start your zmarkdown server  `npm run server`
+ 2. Send a `POST` request to `http://localhost:27272/{endpoint}`
+ 
+`pm2 monit`: provides a realtime dashboard that fits directly into your terminal, is a [simple way to monitor][pm2-monit] the ressource usage of you server.
 
-Only `metadata` is described in the **Response** sections below.
+[pm2-monit]:http://pm2.keymetrics.io/docs/usage/monitoring/
 
-# Endpoints
+### Limit the ressource usage of your server
 
-## `/epub` - Markdown to HTML
+You can change the number of thread (default: `3`) and the max-memory of each thread (default: `150M`) in you package.json to `scripts.server` :
 
-### Request
+```json
+    "server": "pm2 start -f server/index.js -i 3 --max-memory-restart 150M",
+```
 
-* `opts.images_download_dir`: see `/latex`
-* `opts.local_url_to_local_path`: see `/latex`
+## Requests
+All endpoints respond to `HTTP POST`. The request body must be a JSON and `md` is required in every endpoints, `opts` object depends of endpoint, each endpoint have their own options.
 
-### Response
+### URL
 
-* `metadata.disableToc`, bool
+```
+POST http://localhost:27272/{endpoint}
+```
 
-  Whether or not the input Markdown did **not** contain headings (`#`, `##`, …). This property is named that way because we use it to disable Table of Contents generation when no headings were found.
-  `disableToc: true` means *no headings*
-  `disableToc: false` means at least one *heading*
+### Body
 
-## `/html` - Markdown to HTML
+#### Required Body JSON Value
 
-### Request
+| Name | Type | Description |
+| - | - | - |
+| `md` | string | markdown source string. |
 
-* `opts.disable_ping`, bool, default: `false`
+#### Optionnal Body JSON Value
 
-  [pings][ping] won't get parsed
+| Name | Type | Description |
+| - | - | - |
+| `opts` | JSON | Options, specific option for each endpoints. This part will be supplemented in **Request** of the endpoints sections. |
 
-* `opts.disable_jsfiddle`, bool, default: `false`
+### Response
 
-  JSFiddle [iframes][iframes] are disabled
+All endpoints return `[contents, metadata, messages]` as JSON.
 
-* `opts.inline`, bool, default: `false`
+| Name | Type | Description |
+| - | - | - |
+| `contents` | string | the rendered HTML or LaTeX. |
+| `metadata` | object | depends on request options. This part will be supplemented in **Response** of the endpoints sections. |
+| `messages` | string[] | info/debug/errors from parsers, plugins, compilers, etc. |
 
-  Only parse inline Markdown elements (such as links and emphasis, unlike lists and fenced code blocks)
+Only `metadata` is described in the **Response** sections below.
 
-* `opts.stats`, bool, default: `false`
+### Example
```

Comment:
This section is a bit confusing, I suggest removing it. There already is an example with `curl` above.